consensus/consensus_validator.go

```diff
@@ -12,6 +12,29 @@ import (
 	"github.com/harmony-one/harmony/internal/utils"
 )
 
+func (consensus *Consensus) sendBFTBlockToStateSyncing(consensusID uint32) {
+	// validator send consensus block to state syncing
+	if val, ok := consensus.blocksReceived[consensusID]; ok {
```

Comment:
golang will usual check !ok and return early.

Reply:
This will also return if !ok